multi-node/libvirt/network-kubernetes.xml

```diff
@@ -0,0 +1,14 @@
+<network>
+  <name>kubernetes</name>
+  <uuid>dcbef342-16a3-4f3e-8307-84bff91076e1</uuid>
+  <forward mode='nat'/>
+  <bridge name='virbrk8s' stp='on' delay='0'/>
+  <mac address='82:84:01:56:df:f3'/>
+  <ip address='192.168.120.1' netmask='255.255.255.0'>
+    <dhcp>
+      <range start='192.168.120.2' end='192.168.120.254'/>
+      <host mac='54:52:00:fe:b3:b0' name='kube-master' ip='192.168.120.10'/>
+      <host mac='54:52:00:fe:b3:b1' name='kube-worker' ip='192.168.120.11'/>
```

Comment:
really minor nit: maybe leave some ip span between masters/workers. In the case of multi-master / multi-worker clusters it makes it easier to immediately see the roles from a network perspective. But this may just be my laziness...